### ISSUE TITLE
Add aria-label to icons

### ISSUE THREAD

**Issue by josh18** [josh18/TinyMCE-FontAwesome-Plugin#14]
For accessibility. Should be able to get labels from the icons.yml file.

**Comment by josh18** [comment]
Closing as there is not really a good way to source the labels.
